modules/motions/utils/validateAddress.ts

```diff
@@ -0,0 +1,11 @@
+import { constants, utils } from 'ethers'
+
+export function validateAddress(value: string) {
+  if (!utils.isAddress(value)) {
+    return 'Address is not valid'
+  }
+
+  if (value.toLowerCase() === constants.AddressZero) {
+    return 'Address must not be zero address'
+  }
+}
```

Comment:
What do you think about [consistent-return](https://eslint.org/docs/latest/rules/consistent-return)?